projects/schematics/src/migrations/2_0/constructor-deprecation-data.ts

```diff
@@ -141,4 +148,64 @@ export const CONSTRUCTOR_DEPRECATION_DATA: ConstructorDeprecation[] = [
       },
     ],
   },
+  // projects/storefrontlib/src/cms-components/myaccount/address-book/address-book.component.service.spec.ts
+  {
+    class: ADDRESS_BOOK_COMPONENT_SERVICE,
+    deprecatedParams: [
+      {
+        className: FEATURE_CONFIG_SERVICE,
+        importPath: SPARTACUS_CORE,
+      },
+    ],
+    removeParams: [
+      {
+        className: FEATURE_CONFIG_SERVICE,
+        importPath: SPARTACUS_CORE,
+      },
+    ],
+    addParams: [
+      {
+        className: CHECKOUT_DELIVERY_SERVICE,
+        importPath: SPARTACUS_CORE,
+      },
+    ],
+  },
+  // projects/storefrontlib/src/cms-components/checkout/guards/checkout.guard.ts
+  {
+    class: CHECKOUT_GUARD,
+    deprecatedParams: [
```

Comment:
In here you need to specify all currently required services: In case of checkout guard that will be: Router, CheckoutConfig and RoutingConfigService.

Reply:
Same applies to config above.